now.json

```diff
@@ -0,0 +1,19 @@
+{
+  "redirects": [
+    { "source": "/about\\.html", "destination": "/about" },
+    { "source": "/foundation/", "destination": "/about#foundation" },
```

Comment:
Does this also work for for urls without trailing slash?

Reply:
No. But I think URLs without the slash should not exist, because it is added automatically.